Test/TestConfigurationUtils.ps1

```diff
@@ -231,13 +224,12 @@ function Test-IfVmSwitchExist {
 
 function Write-IpAddresses {
     Param (
-        [Parameter(Mandatory = $true)] [PSSessionT] $Session,
-        [Parameter(Mandatory = $true)] [SystemConfig] $SystemConfig
+        [Parameter(Mandatory = $true)] [Testbed] $Testbed
     )
 
-    $AdaptersNames = @($SystemConfig.VHostName, $SystemConfig.AdapterName)
+    $AdaptersNames = @($Testbed.GetVHostName(), $Testbed.DataAdapterName)
```

Comment:
It's inconsistent, we should decide which approach to we prefer. Personally, I lean towards getters and setters.

Reply:
As we talked IRL I will rewrite the fields to be set in Constructor.